### ISSUE TITLE
Large `funded-accounts.json` causes invalid chain spec

### ISSUE THREAD

**Issue by ggwpez** [paritytech/polkadot-stps#24]
When using a `funded-accounts.json` file with 100k accounts, the following error can be observed.  
Not sure what is going on. I personally know that JS has a string length limit of 512MiB above the JS tooling becomes worthless, but we are currently far from that limit.

```pre
Error: Invalid chain spec raw file generated.
    at /snapshot/zombienet/dist/providers/k8s/chain-spec.js:102:19
    at Generator.next (<anonymous>)
    at fulfilled (/snapshot/zombienet/dist/providers/k8s/chain-spec.js:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by pepoviola** [comment]
Hi @bernardoaraujor this is still needed for k8s provider right? I can check if needed.
Thanks!

**Comment by bernardoaraujor** [comment]
hey @pepoviola thanks for checking in on this.

I did some tests zombienet `v1.3.52` and it seems that there were some improvements.
I was able to successfully load up to 100k prefunded accounts, which was bigger than the 72k from my last attempt last year.

I'm not sure what the exact limit is, but it failed againafter trying 300k.

Anyways, I'd say this is not really high priority right now, since the current work of @bredamatt with Versi uses a different method for loading prefunded accounts.

I'll close the issue for now, and re-open in the future if we need to improve this again.
